public/docs/_examples/architecture/dart/lib/sales_tax_component.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:angular2/core.dart';
+
+import 'sales_tax_service.dart';
+import 'tax_rate_service.dart';
+
+@Component(
+    selector: 'sales-tax',
+    template: '''
+      <h2>Sales Tax Calculator</h2>
+      Amount: <input #amountBox (input)="0" type="number">
```

Comment:
What triggered this change (also in TS)? I am not sure I like `(input)` as the event name. I don't know, I guess it is just me but I liked `(change)` more. Input is also the element name so as an event again is weird. Also `input` sounds more like an input (pun intended) than as an output.

On the other hand, the number input is really broken in other browsers. I don't think they are fixed yet.